spring-batch-core/src/main/java/org/springframework/batch/core/step/builder/AbstractTaskletStepBuilder.java

```diff
@@ -74,6 +74,25 @@ public AbstractTaskletStepBuilder(StepBuilderHelper<?> parent) {
 		super(parent);
 	}
 
+	/**
+	 * Create a new builder initialized with any properties in the parent. The parent is
+	 * copied, so it can be re-used.
+	 * @param parent a parent helper containing common step properties
+	 *
+	 * @author Ilpyo Yang
```

Comment:
Maybe `@author` should be located on `L51`, not here?

Reply:
change to ```L51``` completed!